tests/test_database_matrix.py

```diff
@@ -101,45 +101,45 @@ async def test_default_config(patched_send, opsdroid_matrix):
     )
 
 
-@pytest.mark.asyncio
```

Comment:
was this causing trouble?

Reply:
Yeah

Reply:
what was the issue?

Reply:
It was intermittently failing for me locally. But it was a while ago now so I can't remember all the details. It may have been a red herring.

Reply:
I re-enabled the test. We'll see if it is still flaky now that I've separated the signal tests.